rules/windows/powershell/powershell_script/posh_ps_malicious_keywords.yml

```diff
@@ -35,9 +37,7 @@ detection:
             - 'TOKEN_INFORMATION_CLASS'
             - 'TOKEN_PRIVILEGES'
             - 'TOKEN_QUERY'
-            - 'Metasploit'
-            - 'Mimikatz'
     condition: selection
 falsepositives:
-    - Unknown
+    - Depending on the scripts, this rule might require some initial tunning to fit the environment
```

Comment:
then it shouldn't be level high, should it? What were the detected FPs?

Reply:
This is the threat hunting version of the rule with no filter. The detection version is `f742bde7-9528-42e5-bd82-84f51a8387d2` and it has additional filters and is level high.

Reply:
Yes it shouldn't. The FP was some legitimate MS script that has the keywords `LSA_UNICODE_STRING` which is the same used by Mimikatz. But the string isn't unique to MIMIKATZ but its a legit struct name. I'll reduce it to medium.